service/src/main/kotlin/xyz/lebkuchenfm/external/storage/repos/XSoundsMongoRepository.kt

```diff
@@ -14,13 +14,30 @@ class XSoundsMongoRepository(database: MongoDatabase) : XSoundsRepository {
     override suspend fun findAll(): List<XSound> {
         return collection.find().map { it.toDomain() }.toList()
     }
+
+    override suspend fun insert(sound: XSound) {
+        collection.insertOne(XSoundEntity(sound))
+    }
 }
 
 data class XSoundEntity(
-    @BsonId val id: ObjectId,
+    @BsonId val id: ObjectId? = null,
```

Comment:
Wait, why null? ObjectId cannot be null, all entires in our database have `_id` field 🤔

Reply:
Because we have to insert entity to database first and only after that we get id.

Reply:
Okay, I might be interesting to follow up later 👍